vamb/__main__.py

```diff
@@ -222,41 +222,41 @@ def __init__(
 
 class TaxonomyOptions:
     __slots__ = [
-        "taxonomy_path",
-        "taxonomy_predictions_path",
         "no_predictor",
+        "taxonomy_path",
+        "column_contigs",
+        "column_taxonomy",
+        "delimiter_taxonomy",
     ]
 
     def __init__(
         self,
-        taxonomy_path: Optional[Path],
-        taxonomy_predictions_path: Optional[Path],
         no_predictor: Optional[bool],
+        taxonomy_path: Optional[Path],
+        column_contigs: Optional[str],
+        column_taxonomy: Optional[str],
```

Comment:
Not a big deal, but since these must be either both None, or both str, can you check this in `__init__`? Otherwise, a user might pass only, say `--column_contigs` and be confused when it's ignored.